### ISSUE TITLE
Keybinding ("A") for selecting all points in the current view slice crashes napari

### ISSUE THREAD

**Issue by adamltyson** [napari/napari#1352]
## 🐛 Bug

When selecting points, using the keybinding "A" (Select all points in the current view slice), crashes napari.

## To Reproduce

```bash
conda create --name naparibugtest python=3.7
conda activate naparibugtest
pip install napari[all]
python
```
```python
import napari
with napari.gui_qt():
    napari.view_points()
```
If you then choose the point selecting tool and press "A", you get:
```python
  File "/home/adam/miniconda3/envs/naparibugtest/lib/python3.7/site-packages/napari/layers/points/_points_key_bindings.py", line 58, in select_all
    layer.selected_data = layer._indices_view[: len(layer._data_view)]
AttributeError: 'Points' object has no attribute '_data_view'
Aborted (core dumped)
```

## Expected behavior

To select all the points

## Environment
```
napari: 0.3.4
Platform: Linux-4.18.2-041802-generic-x86_64-with-debian-buster-sid
Python: 3.7.7 (default, May 7 2020, 21:25:33) [GCC 7.3.0]
Qt: 5.14.2
PyQt5: 5.14.2
NumPy: 1.18.5
SciPy: 1.4.1
Dask: 2.18.1
VisPy: 0.6.4

GL version: 4.6.0 NVIDIA 418.67
MAX_TEXTURE_SIZE: 32768

Plugins:
- napari-plugin-engine: 0.1.6
- svg: 0.1.3

**Comment by sofroniewn** [comment]
That's a bug! I just reproduced. Should be `_view_data` instead of `_data_view` we swapped the name a while back. Can you test/ submit a PR - ideally with that swap and a minimal test that would have caught the bug.

**Comment by sofroniewn** [comment]
I have implemented a fix in #1358
